fecgov/openFEC
# [Merge after 4095 due to migration file order] Fix candidate_search column to return all possible results

## Summary (required)
Modified the MV to concat the cand_fulltxt tsvector column results.
- Resolves #4086 

_Include a summary of proposed changes._
- updated the `real_efile_se_f57_vw`

## How to test the changes locally
Run steps 1 - 3 to make sure the new migration file complies against your local `cfdm_test` db 
without throwing any migration errors.

1.  run `dropdb cfdm_test`
2.  run `createdb cfdm_test`
3.  run `flyway migrate`
4.  run `pytest`
5. Test against a tmp view thats created in dev db: 
6. In itemized.py/SE efile model, change the table to point to the tmp view` real_efile_se_f57_vw_tmp_pk` here:
https://github.com/fecgov/openFEC/blob/feature/4086-fix-sche_e-efile-cand-search/webservices/common/models/itemized.py#L660
7. export `SQLA_CONN` to point to `DEV` db
8. run server & test the `schedules/schedule_e/efile/` endpoint
Example : `candidate_search = love`
This call will return 45 results:
 http://127.0.0.1:5000/v1/schedules/schedule_e/efile/?sort_hide_null=false&candidate_search=love&sort_nulls_last=false&page=1&sort=-expenditure_date&sort_null_only=false&per_page=20

   In prod, returns only 44 results:
https://api.open.fec.gov/v1/schedules/schedule_e/efile/?api_key=DEMO_KEY&sort_hide_null=false&sort_nulls_last=true&data_type=efiling&candidate_search=love&is_notice=true&sort=expenditure_date&per_page=50&page=1

## Impacted areas of the application

-  Independent Expenditure Raw filings
https://www.fec.gov/data/independent-expenditures/?data_type=efiling&is_notice=true
